src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -221,6 +249,145 @@ public static FamilyType ByName(string name)
             };
         }
 
+        /// <summary>
+        /// Create new Family Instance from Geometry (via SAT import)
+        /// </summary>
+        /// <param name="geometry"></param>
+        /// <param name="name"></param>
+        /// <param name="category"></param>
+        /// <param name="templatePath"></param>
+        /// <param name="isVoid"></param>
+        /// <param name="material"></param>
+        /// <param name="subcategory"></param>
+        /// <returns></returns>
+        public static FamilyType ByGeometry(Autodesk.DesignScript.Geometry.Geometry geometry, string name, Category category, string templatePath, Material material, bool isVoid = false, string subcategory = "")
+        {
+            Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.ForceCloseTransaction();
```

Comment:
The wrapper is a singleton. Right now there is an undocumented rule that everything should be completed in one transaction.  If you insist to implement in this way, at least all Revit test cases need to be verified to pass.